lib/Common/CommonPal.h

```diff
@@ -637,9 +648,9 @@ namespace PlatformAgnostic
 #endif
     }
 
-    __forceinline unsigned char _BitTest(const LONG *_BitBase, int _BitPos)
+    __forceinline unsigned char _BitTest(LONG *_BitBase, int _BitPos)
```

Comment:
Why was the const removed here? seems like allowing for const can result in better codegen by the compiler- if it's android specific, I'd prefer doing a const_cast when _bittest is called scoped to Android